connmgr/connmanager_test.go

```diff
@@ -5,14 +5,21 @@
 package connmgr
 
 import (
+	"fmt"
+	"github.com/kaspanet/kaspad/addrmgr"
+	"github.com/kaspanet/kaspad/config"
+	"github.com/kaspanet/kaspad/dagconfig"
 	"github.com/pkg/errors"
 	"io"
+	"io/ioutil"
 	"net"
 	"sync/atomic"
 	"testing"
 	"time"
 )
 
+const numAddressesInAddressManager = 10
```

Comment:
Are we absolutely positive this is going to be the same value for all tests for ever and ever?

I think it's better if every test decides for itself how much addresses it wants in the address manager

Reply:
Just note that if we implement this you must make sure addressManagerForTest supports arbitrary limits (right now it can't support more then 10) + fails if you are above what is supported.